services/api-server/src/simcore_service_api_server/models/domain/groups.py

```diff
@@ -0,0 +1,15 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+
+class UsersGroup(BaseModel):
+    gid: str
+    label: str
+    description: Optional[str] = None
+
+
+class Groups(BaseModel):
+    me: UsersGroup
+    organizations: Optional[List[UsersGroup]] = []
+    all_: UsersGroup = Field(..., alias="all")
```

Comment:
could we move these things down into the postgres package instead? or else we get tons of duplicates.

Reply:
yes, probably between a separate package and simcore_package_postgres